exercises/concept/weighing-machine/.docs/instructions.md

```diff
@@ -2,99 +2,72 @@
 
 In this exercise you'll be modelling a weighing machine.
 
-The weight can be set and retrieved in pounds or kilograms and cannot be negative.
+You have 6 tasks each of which requires you to implement one or more properties:
 
-The weight can be displayed in SI units or US units
-, pounds and ounces.
+## 1. Allow the weighing machine to have a precision.
 
-A tare adjustment can be applied to the weight (for instance to deduct the
-weight of a container). This can be any value (even negative or a value that makes the display weight negative)
-as there are doubts about the accuracy
-of the weighing machine. For security reasons this value cannot be retrieved.
-
-Note that:
-
-```
-display-weight = input-weight - tare-adjustment
-```
-
-Conversion ratios are as follows:
-
-- 16 ounces to a pound
-- 2.20462 kg to a pound
-
-For Example:
-
-- 60 kilograms == 132.2772 ponds
-- 132.2772 pounds == 132 pounds 4 ounces
-
-You have 5 tasks each of which requires you to implement one or
-more properties:
-
-## 1. Allow the weight to be set on the weighing machine
-
-Implement the `WeigingMachine.InputWeight` property to allow the weight to be get and set:
+It important to ensure that our weighing machine is precise.
```

Comment:
To cater to different demands, we allow each weighing machine to be customized with a precision (the number of digits  after the decimal separator).